c/meterpreter/source/metsrv/pool_party_ext.h

```diff
@@ -0,0 +1,585 @@
+#ifndef POOL_PARTY_H
+#define POOL_PARTY_H
+#pragma once
```

Comment:
This is redundant with the `#pragma once` no?

Reply:
I was getting some compilation error but maybe it get fixed moving around. gonna try.